### ISSUE TITLE
Permission for last location for /back

### ISSUE THREAD

**Issue by cryoten** [Zrips/CMI#3818]
**Description of feature/improvement request:**

From my understanding, `/back` automatically saves your previous location when you teleport and the permission `cmi.command.back.ondeath` allows your death location to be added as a previous location too.

It would be nice to change that slightly so that we can disable the last teleport location from being saved.

Possible additional permission `cmi.command.back.onlastlocation`

---
### Detailed explanation of what it should do, when and in what situation: ###

When both `onlastlocation` and `ondeath` are false but `cmi.command.back` is true, players can use the command `/back` but there are is no previous location saved so it spits out an error message in the chat to inform the player.

When `ondeath` is true but `onlastlocation` is false, only your death location is saved so you can go `/back` to the spot where you died.

When `onlastlocation` is true, all previous locations from teleports get saved as is currently the case. Defaults to true so that current functionality remains and this is merely an option that admins can turn off.

### Basic explanation as to why this feature is needed and/or useful to have: ###

This suggestion would allow server admins to modify what `/back` does. For example, a player of low rank may only be able to travel back to their death location while a player with a higher rank could travel back to not only their death location, but also their last location.

**Comment by CloudeLecaw** [comment]
you know you also can give permission to just teleport back to you last death location with dback?

**Comment by cryoten** [comment]
Ahhh, I was not familiar with dback. Thanks!

**Comment by CloudeLecaw** [comment]
Dont forget to close the request if thus allready fullfill your needs :)

**Comment by cryoten** [comment]
I ended up not using it. This would work great for rank 1 players but for rank 3 I wanted them to have normal use of `back`. I was able to disable the custom alias and enable `/cmi back` but I wasn't sure how to have an alias for it when there was a custom one already in place.

Sure, I could give them permission to use `/dback` then remove it when they get `/back` but that adds to the confusion so I avoided doing that.

Unless you know of a way to allow the custom alias to use `/cmi dback` when they are rank 1, and `/cmi back` when they are rank 3, then I'd still like a permission added. :)
